Makefile

```diff
@@ -205,7 +205,12 @@ generate-go: $(MOCKGEN)
 		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt
 
 	$(CONVERSION_GEN) \
-		--input-dirs=./api/v1beta1 \
+		--input-dirs=./api/v1beta3 \
```

Comment:
I didn't pay attention to this before
curious on v1beta3/v1beta4 comes from ? just for learning purpose ..

Reply:
Sorry these should be v1alpha3 and v1alpha4. I will update.